src/riak_pb_ts_codec.erl

```diff
@@ -124,6 +124,8 @@ encode_cell(V) when is_boolean(V) ->
 encode_cell({time, V}) ->
     #tscell{timestamp_value = V};
 encode_cell(undefined) ->
+    #tscell{};
+encode_cell([]) -> %% NULL Cell
```

Comment:
So the issue I found is within the generated ddl modules, they only accept `[]` for a null cell value. I think it used to be `undefined` but not sure if something happened.